ocean_lib/ocean/crypto.py

```diff
@@ -18,7 +18,7 @@
 def calc_symkey(base_str: str) -> str:
     """Compute a symmetric private key that's a function of the base_str"""
     base_b = base_str.encode("utf-8")  # bytes
-    hash_b = sha256(base_b)
+    hash_b = sha256(base_b).hexdigest()
     symkey_b = b64encode(str(hash_b).encode("ascii"))[:43] + b"="  # bytes
```

Comment:
I think in line 22, we no longer need the str cast, since hash_b is already a string